packages/ref-doc/src/utils/markdown.ts

```diff
@@ -23,7 +23,7 @@ export function inlinecode(code: string) {
 }
 
 function escapeMarkdownTable(text: string) {
-  return text.replace(/(\|)/g, "\\$1");
+  return text.replace(/(\|)/g, "\\$1").replace(/\n/g, "<br />");
```

Comment:
Pedantically, `<br />` wastes bytes over `<br>`, the self-closing tag syntax is an xhtml holdover that isn't necessary in HTML5 (which, if you're curious, always ignores trailing whitespace and slashes, and maintains a list of "void elements" that don't need any closing syntax for things like `img` and `br`).

Reply:
Continuing aside - this results in the annoyance that custom elements can't ever be self-closing, i.e. `<my-custom-element />` is valid syntax but is semantically identical to `<my-custom-element>`, i.e. it is unclosed.

Reply:
I will double check, but I think it barfed at me when I did `<br>` first.

Reply:
This auto-merged before I could double check, but indeed, something in the stack does not like `<br>`.

```
Compiled with problems:X

ERROR in ../../docs/compiler/reference/decorators.md

Module build failed (from ../../common/temp/node_modules/.pnpm/@docusaurus+mdx-loader@2.3.0_de61d8f3d8ac64703c5df6ba8f9598df/node_modules/@docusaurus/mdx-loader/lib/index.js):
SyntaxError: C:\Src\cadl\docs\compiler\reference\decorators.md: Expected corresponding JSX closing tag for <br>. (295:96)
```

Reply:
Ohhhhh because this is JSX (an html-like syntax embedded in JS) not html. Interesting, ok. Yeah jsx is more xhtml like.

Reply:
Ohhhhh because this is JSX (an html-like syntax embedded in JS) not html. Interesting, ok. Yeah jsx is more xhtml like.

Reply:
Ah, yes, I totally understood this when I made this change, and didn't like blindly futz with it until it worked. 🤣